publish/skylink.debug.js

```diff
@@ -5301,6 +5301,15 @@ Skylink.prototype._setLocalAndSendMessage = function(targetMid, sessionDescripti
     sessionDescription.sdp = sessionDescription.sdp.replace(/a=rtcp-fb:100 goog-remb\r\n/g, '');
   }
 
+  // Remove rtx or apt= lines that prevent connections for browsers without VP8 or VP9 support
+  // See: https://bugs.chromium.org/p/webrtc/issues/detail?id=3962
+  if (['chrome', 'opera'].indexOf(window.webrtcDetectedBrowser) > -1) {
```

Comment:
Instead of hard fixing Chrome 50 and Opera 37 to Safari/IE browsers, it seems like for older versions to Firefox had this issue as well. See: https://bugs.chromium.org/p/webrtc/issues/detail?id=3962

Since these payload causes issues, might be better to remove them.